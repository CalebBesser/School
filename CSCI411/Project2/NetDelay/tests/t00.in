6
addEdge 0 1 4
addEdge 0 2 5
addEdge 2 3 7
addEdge 2 4 4
addEdge 4 5 4
printGraphW
findNetDelay
