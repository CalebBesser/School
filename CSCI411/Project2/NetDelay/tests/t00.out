Graph is:
0: 1/4	2/5	
1: 0/4	
2: 0/5	3/7	4/4	
3: 2/7	
4: 2/4	5/4	
5: 4/4	

The longest path is:
1 0 2 4 5 
The length of the longest path is 17.
